api/document/serializers/util.py

```diff
@@ -43,3 +45,16 @@ def iter_inlines(inlines: List[PrimitiveDict]) -> Iterator[PrimitiveDict]:
     for inline in inlines:
         yield inline
         yield from iter_inlines(inline['inlines'])
+
+
+def list_to_internal_value(data: List[PrimitiveDict],
+                           field_class: Type[Field],
+                           *args, **kwargs) -> List[PrimitiveDict]:
+    # The only real reason this function exists is because
+    # the way DRF's `many=True` changes the type signatures of
+    # a serializer's methods is extremely hard/impossible to
+    # express as a type annotation. So this is really just a
+    # way to "twist mypy's arm" into annotating things the way we
+    # want.
+    serializer = field_class(*args, **{'many': True, **kwargs})
+    return serializer.to_internal_value(data)
```

Comment:
I just realized that given https://github.com/18F/omb-eregs/pull/961#issuecomment-361252229, it might be smarter to do this the "proper" way for `Serializer` subclasses, e .g. by passing `data` to their constructor, then calling `.is_valid()`, then returning `.validated_data`.  If we do that, we should also change the type signature of this function to ensure that `field_class` is really `serializer_class` and is of `Type[Serializer]` (which should be OK, I think that we're only ever passing such objects to this function anyways).

Reply:
I _do_ like propriety.